src/Poll/Poll.model.ts

```diff
@@ -22,6 +31,50 @@ export class Poll extends Model<PollAttributes> {
       ORDER BY p.created_at`)
   }
 
+  static async findWithAssociationsWithFilters(
```

Comment:
I think it might make sense to just accept an object here so:
- you can feed it the result of sanitize
- adding properties becomes easier
- avoids having having to guess what these last parameters are: `findWithAssociationsWithFilters(1, 0, true, false)`